### PR TITLE
Modify ortho_bored config to more closely match Moonlander.

### PR DESCRIPTION
**Pull request by natsume42**
Please check whether my modifications actually make sense. There is a good deal of guesswork involved in my changes.

Intended changes:
- Introduced another thumb key. Updated tables and graphical representation accordingly.
- Reassigned mods, space, and BACKSPACE to thumb keys. No need to define them for both hands.
- Introduced another mod key: ⑤
- In an effort to reduce the high ratio of not found trigrams (see below), I added grave and acute accents to layer 3. But this did not help. 

Observations: 
- Keyboard produces very low result for e.g. : zluaqßbdmwjhrieocntsgfyxüäövp,.k of 136,84, which is surprisingly low.
- There is a suspicious looking output in the evaluation that might hint at some sort of issue. I do not understand it though: "Trigram metrics:  Not found: 6.8399% of 486487579.7860". This is high compared to standard keyboard layout.

### CODE REVIEW

**Comment by dariogoetz** [comment]
I had a little time to take a look at the changes. I think that I understand the effects that you are observing:
1. Low results: This is mostly due to the shift key being on the thumb keys. For several metrics, thumb keys are excluded. This made sense for "classical" keyboard layouts. We would need to re-evaluate, whether that choice still makes sense for ortho keyboards with thumb keys that cover more than just "space".
2. The high number of "not found" trigrams is due to the character "⇗" appearing often in the prepared ngrams by ArneBab that we are using. I am not sure, where they come from, but they are definitely there. See `3-gramme.arne*.txt`.

**Comment by natsume42** [comment]
ad 1) In this case, I would like to suggest to always handle thumb keys as regular keys. Especially, there should be respective cost values associated with them and they should be considered in the metrics.
ad 2) I think this requires further clarification. I could add this "key" somewhere to the layout of course, but without knowing the background of this, adding the key might turn out to be just as "wrong" and cause other unintended side-effects.

File: config/ortho_bored.yml [hunk 1]
```diff
@@ -1,69 +1,69 @@
...
+    ┌─────┬───┤ {{2}} ├───┤ {{4}} │ {{5}} │            │ {{6}} │ {{7}} ├───┤ {{9}} ├───┬─────┐
+    │  {{0}}  │ {{1}} ├───┤ {{15}} ├───┼───┤            ├───┼───┤ {{20}} ├───┤ {{10}} │  {{11}}  │
+    ├─────┼───┤ {{14}} ├───┤ {{16}} │ {{17}} │            │ {{18}} │ {{19}} ├───┤ {{21}} ├───┼─────┤
+    │  {{12}}  │ {{13}} ├───┤ {{27}} ├───┼───┤            ├───┼───┤ {{32}} ├───┤ {{22}} │  {{23}}  │
+    ├─────┼───┤ {{26}} ├───┤ {{28}} │ {{29}} │            │ {{30}} │ {{31}} ├───┤ {{33}} ├───┼─────┤
+    │  {{24}}  │ {{25}} ├───┤ {{39}} ├─━─┼───┤            ├───┼─━─┤ {{44}} ├───┤ {{34}} │  {{35}}  │
+    ├─────┼───┤ {{38}} ├───┤ {{40}} │ {{41}} │            │ {{42}} │ {{43}} ├───┤ {{45}} ├───┼─────┤
+    │  {{36}}  │ {{37}} ├───┘   └───┴───┘            └───┴───┘   └───┤ {{46}} │  {{47}}  │
+    ├─────┼───┼───┬───┬───┐┌───┬───┐  ┌───┬───┐┌───┬───┬───┼───┼─────┤
+    │  {{48}}  │ {{49}} │ {{50}} │ {{51}} │ {{52}} ││ {{53}} │ {{54}} │  │ {{55}} │ {{56}} ││ {{57}} │ {{58}} │ {{59}} │{{60}}  │  {{61}}  │
```
Comment (Glitchy-Tozier) [inline config/ortho_bored.yml:65]: I think something went wrong with the spacing of the `{{60}}`-key. Suggestion:
```diff
- │{{60}}  │
+ │ {{60}} │
```

**Comment by Glitchy-Tozier** [comment]
Maybe we should also rename "ortho_bored" to something that resembles Moonlander or similar keyboards. :)
"ortho_bored" might already hint at something, but I'm not sure what is being implied here.

**Comment by natsume42** [comment]
> Maybe we should also rename "ortho_bored" to something that resembles Moonlander or similar keyboards. :) "ortho_bored" might already hint at something, but I'm not sure what is being implied here.
Yes, I agree. I think Ortho Moonlander would be more to the point. Even though it does not match it perfectly yet.

**Comment by dariogoetz** [comment]
> > Maybe we should also rename "ortho_bored" to something that resembles Moonlander or similar keyboards. :) "ortho_bored" might already hint at something, but I'm not sure what is being implied here.
> 
> Yes, I agree. I think Ortho Moonlander would be more to the point. Even though it does not match it perfectly yet.

I would say that this depends on whether the specific configuration of the thumb cluster can be considered standard (to some extent). If it is something specific to natsume42, then it should keep a "personal" name. Maybe "Moonlander (bored)" or Moonlander (natsume42)". Maybe, we can also consider introducing a "community " subdropdown.

**Comment by Glitchy-Tozier** [comment]
> Maybe "Moonlander (bored)" or...

@dariogoetz What is up with that "bored"? Does that word somehow connect to a specific set of keyboards? I only know of the meaning "gelangweilt".

**Comment by natsume42** [comment]
Well, the thumb cluster is not specific to me. That is the standard configuration of Moonlander keyboard. And I am busy preparing another pull request that covers the last missing piano key.

**Comment by dariogoetz** [comment]
> Well, the thumb cluster is not specific to me. That is the standard configuration of Moonlander keyboard. And I am busy preparing another pull request that covers the last missing piano key.

Then I'm fine with naming it "Moonlander" or "Ortho Moonlander" or the like

**Comment by natsume42** [comment]
I updated the branch. It now includes all 3 piano keys (per pad).

**Comment by natsume42** [comment]
Seems you already merged my previous state, so I will create a new pull request for the additional piano keys. Hope that I will be able to handle the merge correctly…

**Comment by dariogoetz** [comment]
I did not merge anything, yet. I only fixed the bug for the positions in the lowest row (also for Ortho.yml)

**Pull request closed without merging**